repository/src/main/java/com/epam/esm/repository/builder/OrderQueries.java

```diff
@@ -0,0 +1,10 @@
+package com.epam.esm.repository.builder;
+
+public final class OrderQueries {
```

Comment:
Why did you decide to use separate immutable classes to store queries to each entity?